sklearn/utils/validation.py

```diff
@@ -155,6 +156,32 @@ def _shape_repr(shape):
     return "(%s)" % joined
 
 
+def check_memory(memory):
+    """Check that the memory is an instance of joblib.Memory.
+
+    Raises a ValueError if the passed object does not have a
+
+    cache attribute.
+
+    Parameters
+    ----------
+    memory: input object.
+
+    Returns
+    -------
+    memory: the input memory if it is valid. A valueError if invalid memory instance.
```

Comment:
need space before `:`. The right of the colon should only specify type, i.e. Memory.